generators/typescript/utils/core-utilities/fetcher/src/fetcher/getRequestBody.ts

```diff
@@ -1,34 +1,14 @@
-import { RUNTIME } from "../runtime";
-
-export async function getRequestBody(body: any, contentType: string): Promise<BodyInit | undefined> {
-    let requestBody: BodyInit;
-    if (RUNTIME.type === "node") {
-        if (body instanceof (await import("formdata-node")).FormData) {
-            // @ts-expect-error
-            requestBody = body;
-        } else if (body instanceof (await import("stream")).Readable) {
-            // @ts-expect-error
-            requestBody = body;
-        } else {
-            requestBody = maybeStringifyBody(body, contentType ?? "");
-        }
-    } else {
-        if (body instanceof (await import("form-data")).default) {
-            // @ts-expect-error
-            requestBody = body;
-        } else {
-            requestBody = maybeStringifyBody(body, contentType ?? "");
-        }
+export declare namespace GetRequestBody {
+    interface Args {
+        body: unknown;
+        type: string;
     }
-    return requestBody;
 }
 
-export const maybeStringifyBody = (requestBody: any, contentType: string): Uint8Array | string => {
-    if (requestBody instanceof Uint8Array) {
-        return requestBody;
-    } else if (contentType === "application/x-www-form-urlencoded" && typeof requestBody === "string") {
-        return requestBody;
+export async function getRequestBody({ body, type }: GetRequestBody.Args): Promise<BodyInit | undefined> {
+    if (type.includes("json")) {
```

Comment:
Who passes in `type` here ?

Reply:
I think we need to decide if we want the `getRequestBody` refactor to be part of this PR -- if the answer is `yes` then we'll need to update [Fetcher.invoke](https://github.com/fern-api/fern/blob/b0c54ff7b0955379f91f72a0c06e56c62c5cfa11/generators/typescript/utils/commons/src/core-utilities/fetcher/FetcherImpl.ts#L75-L78) to take the type field. 

Fetcher invoke is called [here](https://github.com/fern-api/fern/blob/b0c54ff7b0955379f91f72a0c06e56c62c5cfa11/generators/typescript/sdk/client-class-generator/src/endpoints/default/GeneratedDefaultEndpointImplementation.ts#L292) for example